agentops/helpers.py

```diff
@@ -85,19 +85,29 @@ def default(o):
             return o.model_dump_json()
         elif hasattr(o, "to_json"):
             return o.to_json()
+        elif hasattr(o, "json"):
+            return o.json()
+        elif hasattr(o, "to_dict"):
+            return o.to_dict()
+        elif hasattr(o, "dict"):
+            return o.dict()
         else:
             return f"<<non-serializable: {type(o).__qualname__}>>"
 
-    def remove_none_values(value):
-        """Recursively remove keys with None values from dictionaries."""
+    def remove_unwanted_items(value):
+        """Recursively remove unwanted items from dictionaries."""
```

Comment:
Can we define what we mean by "unwanted"?

Reply:
Oh in the docstring? Yeah